src/integration-test/java/developerTC/TestAgentEndpointUpdate_05.java

```diff
@@ -37,7 +39,7 @@ public void testServiceEndPoint() {
         assertNull(remoteAgent.getAuthEndpoint());
```

Comment:
Why are we checking for a null auth endpoint? Is this expected?

Reply:
Reverted the changes, will raise PR separately for a separate issue.